src/ontology/aeon-edit.owl

```diff
@@ -1160,15 +1160,14 @@ SubClassOf(:AEON_0000001 obo:OBI_0000011)
 SubClassOf(:AEON_0000001 ObjectSomeValuesFrom(obo:BFO_0000055 :AEON_0000005))
 SubClassOf(:AEON_0000001 ObjectSomeValuesFrom(:AEON_0000043 :AEON_0000030))
 SubClassOf(:AEON_0000001 ObjectSomeValuesFrom(:AEON_0000084 :AEON_0000002))
-DisjointClasses(:AEON_0000001 :AEON_0000002)
```

Comment:
I somehow did not properly refresh the head of this issue branch

(poor excuse: am still fairly new to the GH workflow)